posthog/management/commands/migrate_team.py

```diff
@@ -116,10 +117,10 @@ def handle(self, **options):
                 print()  # noqa: T201
```

Comment:
Not related to this PR but what is this `print` doing here...

Reply:
Guess it's to add a new line in the console output?